sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -456,7 +456,9 @@ private static float setDefaultConfidenceValue(Float confidence) {
             .map(fieldValue -> {
                 FieldData valueData = null;
                 // ARRAY has ho value data, such as bounding box.
-                if (ARRAY != fieldValue.getType()) {
+                if (ARRAY != fieldValue.getType()
```

Comment:
what happens if the field type is not ARRAY only?

Reply:
They will have valueData except for the mentioned two cases.